tgui/packages/tgui/interfaces/AlertComputer.tsx

```diff
@@ -0,0 +1,137 @@
+import { Button, LabeledList } from 'tgui-core/components';
+
+import { useBackend } from '../backend';
+import { Window } from '../layouts';
+
+interface AlertComputerData {
+  alerts: Partial<Record<string, AlertData>>;
```

Comment:
I don't see where the key is really used in this data structure. You're doing length checks on it, but not actually using the `key` string for anything. What does the key of this Record represent?

Reply:
it's used to lazily pack the data on the DM side